todo_app/templates/index.html

```diff
@@ -11,8 +11,21 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items  %}
+        <li> Id: {{item.id}} Title: {{ item.title}} Status: {{item.status}}</li>
+        <form  action="/deleteitem" method ='POST'>
+          <input type =text name= "id" value = "{{item.id}}"/>
```

Comment:
Just tidying up to the normal way HTML is written:
```suggestion
          <input type="text" name="id" value="{{item.id}}"/>
```

Attribute values should be wrapped in `" "` and you don't normally write spaces around the `=`